color-picker/src/commonMain/kotlin/com/godaddy/android/colorpicker/harmony/HarmonyColorPicker.kt

```diff
@@ -33,12 +32,41 @@ import kotlin.math.atan2
 import kotlin.math.hypot
 import kotlin.math.min
 
+@Deprecated(
+    message = "Use version with programmatic updates",
+    replaceWith = ReplaceWith(
+        """HarmonyColorPicker(
+            modifier = modifier,
+            harmonyMode = harmonyMode,
+            value = HsvColor.from(color),
+            onValueChanged = onColorChanged
+        )"""
+    )
+)
 @Composable
 fun HarmonyColorPicker(
     modifier: Modifier = Modifier,
     harmonyMode: ColorHarmonyMode,
     color: Color = Color.Red,
     onColorChanged: (HsvColor) -> Unit
+) {
+    HarmonyColorPicker(
+        modifier = modifier,
+        harmonyMode = harmonyMode,
+        value = HsvColor.from(color),
+        onValueChanged = onColorChanged
+    )
+}
+
+/**
+ * Color Picker that programmatically updates.
+ */
+@Composable
+fun HarmonyColorPicker(
+    modifier: Modifier = Modifier,
+    harmonyMode: ColorHarmonyMode,
+    value: HsvColor = HsvColor.from(Color.Red),
+    onValueChanged: (HsvColor) -> Unit
```

Comment:
We should keep the same naming in both `HarmonyColorPicker` and `ClassicColorPicker`, I don't have a strong opinion about which one to choose, but they must be the same.